python/lsst/ts/phosim/PhosimCmpt.py

```diff
@@ -715,6 +729,79 @@ def getPistonCamStarArgsAndFilesForPhoSim(
 
         return argStringList
 
+    def getWfsStarArgsAndFilesForPhoSim(
+        self,
+        obsId,
+        skySim,
+        simSeed=1000,
+        cmdSettingFileName="starDefault.cmd",
+        instSettingFileName="starSingleExp.inst",
+    ):
+        """Get the star calculation arguments and files for the
+        wavefront sensors for the PhoSim calculation.
+
+        Parameters
+        ----------
+        obsId : int
+            Observation Id.
+        skySim : SkySim
+            Sky simulator
+        cmdFileName : str, optional
+            Physical command file name. (the default is "star.cmd".)
+        instFileName : str, optional
+            Star instance file name. (the default is "star.inst".)
+        logFileName : str, optional
+            Log file name. (the default is "starPhoSim.log".)
+        simSeed : int, optional
+            Random number seed. (the default is 1000)
+        cmdSettingFileName : str, optional
+            Physical command setting file name. (the default is
+            "starDefault.cmd".)
+        instSettingFileName : str, optional
+            Instance setting file name. (the default is "starSingleExp.inst".)
```

Comment:
The order should be `obsId`, `skySim`, `simSeed`, `cmdSettingFileName`, and `instSettingFileName`. In addition, you should not have `cmdFileName`, `instFileName`, and `logFileName`.